loadtest/control/actions.go

```diff
@@ -430,6 +451,7 @@ func SearchPosts(u user.User) UserActionResponse {
 	return UserActionResponse{Info: fmt.Sprintf("found %d posts", len(list.Posts))}
 }
 
+// ViewUser opens a random user profile for the given user.
```

Comment:
I think here it makes sense to write something like:

`ViewUser simulates opening a random user profile for the given user.`